### PR TITLE
RHDEVDOCS-2589

### PR DESCRIPTION
**Pull request by rolfedh**
OpenShift Logging: Update any installation/upgrade specific information 
to use our new channels.
https://issues.redhat.com/browse/RHDEVDOCS-2589
[enterprise-4.7][enterprise-4.8]

### CODE REVIEW

File: modules/cluster-logging-configuring-image-about.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ Promtail is not officially supported at this time.
 ====
 
 ////
-RSYSLOG_IMAGE=registry.redhat.io/openshift4/ose-logging-rsyslog:v4.3 <6>
+RSYSLOG_IMAGE=registry.redhat.io/openshift-logging/rsyslog:v4.3 <6>
```
Comment (jcantrill) [inline modules/cluster-logging-configuring-image-about.adoc:33]: We should be able to remove ref to rsyslog as a collector all-to-gether.  We do not do any active development or support cc @sichvoge

File: modules/cluster-logging-deploy-cli.adoc [hunk 2]
```diff
@@ -139,14 +139,14 @@ metadata:
...
-  channel: "4.7" <2>
+  channel: "5.0" <2>
   installPlanApproval: "Automatic"
   source: "redhat-operators" <3>
   sourceNamespace: "openshift-marketplace"
   name: "elasticsearch-operator"
 ----
 <1> You must specify the `openshift-operators-redhat` Namespace.
-<2> Specify `4.7` as the channel.
+<2> Specify `5.0` as the channel.
```
Comment (jcantrill) [inline modules/cluster-logging-deploy-cli.adoc:149]: Do we also need to mention `stable` as an option here and what that actually means: latest 5.x released code?

File: modules/cluster-logging-log-store-status-comp.adoc [hunk 3]
```diff
@@ -149,7 +149,7 @@ The output includes the following status information:
 ....
   Containers:
    elasticsearch:
-    Image:      registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.3
+    Image:      registry.redhat.io/openshift-logging/elasticsearch-rhel8-operator:latest
```
Comment (jcantrill) [inline modules/cluster-logging-log-store-status-comp.adoc:152]: this is pointing to the operator not the elasticsearch6 images

File: modules/cluster-logging-configuring-image-about.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/cluster-logging-configuring-image-about.adoc:33]: Removed. That doc was commented out cruft.

File: modules/cluster-logging-deploy-cli.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/cluster-logging-deploy-cli.adoc:149]: @jcantrill Replaced it with `Specify `5.0` or `stable` as the channel. `stable` is the latest 5.x released code.`

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Comment by ahardin-rh** [comment]
LGTM!

**Comment by rolfedh** [comment]
@sichvoge  FYI

**Pull request merged**